spec/lib/sidekiq/form526_backup_submission_process/processor_spec.rb

```diff
@@ -47,5 +47,50 @@
         .new(submission.id, get_upload_location_on_instantiation: false)
         .choose_provider(auth_headers, ApiProviderFactory::API_PROVIDER[:lighthouse])
     end
+
+    it 'pulls from the correct Lighthouse provider according to the startedFormVersion' do
+      allow_any_instance_of(LighthouseGeneratePdfProvider).to receive(:generate_526_pdf).and_return(Faraday::Response.new(
```

Comment:
Huh I would have thought the linter would be cranky about this line being too long but it looks like it passed?

Reply:
rubocop caught up with me later